src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java

```diff
@@ -49,12 +49,6 @@
  */
 public abstract class AbstractLoader
     extends DefaultHandler {
-    /** Feature that enables loading external DTD when loading XML files. */
-    private static final String LOAD_EXTERNAL_DTD =
-        "http://apache.org/xml/features/nonvalidating/load-external-dtd";
```

Comment:
@romani  it's kinda troublesome. I can exclude `addFeauresForVerySecureJavaInstallations` from being mutated, but I cannot suppress mutation of _the call_ to this method. `avoidCallTo` in PIT config didn't help, cause it allows only class names as params.
I can pull this method to the separate class, eg `AbstractLoaderUtils`, and this helps. Is it fine?